packages/api/src/external/ehr/athenahealth/command/get-patients-from-appointments.ts

```diff
@@ -187,7 +182,7 @@ async function getAppointmentsAndCreateOrUpdatePatient({
       })
     );
   } catch (error) {
-    const msg = `Failed to get appointments. cxId ${cxId} practiceId: ${practiceId} departmentId: ${departmentId}. Cause: ${errorToString(
+    const msg = `Failed to get appointments. cxId ${cxId} practiceId ${practiceId} departmentIds ${departmentIds}. Cause: ${errorToString(
```

Comment:
ditto re logs

Reply:
same this as above: This is for sentry. The log message would appear in AWS logs, but not sentry. Normally you'd pass an "extras" object with these values (and exclude them from the msg) but I'm aggregating all the errors into a single sentry alert, so I need to make sure they're in the message itself.

Reply:
This one still needs the detailed info as the incoming log is the generic global one.